### ISSUE TITLE
Accent popup on MacOS inserts replicated characters

### ISSUE THREAD

**Issue by yadayadaydadaa** [laurent22/joplin#3393]
In the editor, the accent popup (holding a key pressed and then selecting an accented character by pressing its number) on MacOS inserts duplicated characters, one without the accent, following by the accented character. Only the latter should be inserted.

## Environment

Joplin version: 1..0.218
Platform: MacOS 10.15.4

## Steps to reproduce

1. Edit a note
2. Hold down e.g. `a` (on an appropriate keyboard layout).
3. On the accent popup appears, press, e.g. `1`. (Exact number that it makes sense to press may depend on the  keyboard layout.)
4. Observe `aà` being inserted instead of `à`. (Exact accented character depends on step 3.)

**Comment by laurent22** [comment]
On which editor?

**Comment by tessus** [comment]
This issue has already been opened and is a duplicate of #2959

**Comment by tessus** [comment]
Duplicate of #2959
